feature-wallet-connect-impl/src/main/java/io/novafoundation/nova/feature_wallet_connect_impl/presentation/sessions/list/WalletConnectSessionsViewModel.kt

```diff
@@ -33,6 +35,14 @@ class WalletConnectSessionsViewModel(
         router.openSessionDetails(WalletConnectSessionDetailsPayload(item.sessionTopic))
     }
 
+    private fun sessionsFlow(): Flow<List<WalletConnectSession>> {
+        return if (walletConnectSessionsPayload.metaId == null) {
+            interactor.activeSessionsFlow()
+        } else {
+            interactor.activeSessionsFlow(walletConnectSessionsPayload.metaId)
```

Comment:
Maybe we can just create a single method in interface that will accept optional `metaId` parameter?